Mathlib/NumberTheory/LSeries/HurwitzZetaEven.lean

```diff
@@ -395,3 +407,406 @@ lemma completedCosZeta₀_one_sub (a : UnitAddCircle) (s : ℂ) :
+    simpa only [Gammaℝ_one, inv_one, mul_one] using (completedHurwitzZetaEven_residue_one a).mul
+      <| (differentiable_Gammaℝ_inv.continuous.tendsto _).mono_left nhdsWithin_le_nhds
+  refine this.congr' ?_
+  filter_upwards [eventually_ne_nhdsWithin one_ne_zero] with s hs
+  simp_rw [hurwitzZetaEven_def_of_ne_or_ne (Or.inr hs), mul_div_assoc]
+
+lemma differentiableAt_hurwitzZetaEven_sub_one_div (a : UnitAddCircle) :
+    DifferentiableAt ℂ (fun s ↦ hurwitzZetaEven a s - 1 / (s - 1) / Gammaℝ s) 1 := by
+  suffices DifferentiableAt ℂ
+      (fun s ↦ completedHurwitzZetaEven a s / Gammaℝ s - 1 / (s - 1) / Gammaℝ s) 1 by
+    apply this.congr_of_eventuallyEq
+    filter_upwards [eventually_ne_nhds one_ne_zero] with x hx
+    rw [hurwitzZetaEven, Function.update_noteq hx]
+  simp_rw [← sub_div, div_eq_mul_inv _ (Gammaℝ _)]
+  refine DifferentiableAt.mul ?_ differentiable_Gammaℝ_inv.differentiableAt
+  simp_rw [completedHurwitzZetaEven_eq, sub_sub, add_assoc]
+  conv => enter [2, s, 2]; rw [← neg_sub, div_neg, neg_add_self, add_zero]
+  exact (differentiable_completedHurwitzZetaEven₀ a _).sub
+    <| (differentiableAt_const _).div differentiableAt_id one_ne_zero
+
+/-- Expression for `hurwitzZetaEven a 1` as a limit. (Mathematically `hurwitzZetaEven a 1` is
+undefined, but our construction assigns some value to it; this lemma is mostly of interest for
+determining what that value is). -/
+lemma tendsto_hurwitzZetaEven_sub_one_div_nhds_one (a : UnitAddCircle) :
+    Tendsto (fun s ↦ hurwitzZetaEven a s - 1 / (s - 1) / Gammaℝ s) (𝓝 1)
+    (𝓝 (hurwitzZetaEven a 1)) := by
+  simpa only [one_div, sub_self, div_zero, Gammaℝ_one, div_one, sub_zero] using
+    (differentiableAt_hurwitzZetaEven_sub_one_div a).continuousAt.tendsto
+
+lemma differentiable_hurwitzZetaEven_sub_hurwitzZetaEven (a b : UnitAddCircle) :
+    Differentiable ℂ (fun s ↦ hurwitzZetaEven a s - hurwitzZetaEven b s) := by
+  intro z
+  rcases ne_or_eq z 1 with hz | rfl
+  · exact (differentiableAt_hurwitzZetaEven a hz).sub (differentiableAt_hurwitzZetaEven b hz)
+  · -- NB. This can be written more tidy with `convert`, but the `convert` version is 3x slower, as
+    -- it spends 877 TC synthesis steps vainly trying to infer `Subsingleton (ℂ → ℂ)`.
+    have (s) : (hurwitzZetaEven a s - hurwitzZetaEven b s) = ((hurwitzZetaEven a s -
+        1 / (s - 1) / Gammaℝ s) - (hurwitzZetaEven b s - 1 / (s - 1) / Gammaℝ s)) := by abel
+    exact funext this ▸ (differentiableAt_hurwitzZetaEven_sub_one_div a).sub
+      (differentiableAt_hurwitzZetaEven_sub_one_div b)
+
+/--
+Formula for `hurwitzZetaEven` as a Dirichlet series in the convergence range, with sum over `ℤ`.
+-/
+lemma hasSum_int_hurwitzZetaEven (a : ℝ) {s : ℂ} (hs : 1 < re s) :
+    HasSum (fun n : ℤ ↦ 1 / (↑|n + a| : ℂ) ^ s / 2) (hurwitzZetaEven a s) := by
+  rw [hurwitzZetaEven, Function.update_noteq (not_lt.mpr zero_le_one <| zero_re ▸ · ▸ hs)]
```

Comment:
This looks like [Complex.ne_zero_of_one_lt_re](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Complex/Abs.html#Complex.ne_zero_of_one_lt_re) again (there may be other instances I have overlooked...).

Reply:
I fixed the two you found and one other – I think that's the lot.